### PR TITLE
Link URL encoding and percent-encoding to relevant place

### PR DESCRIPTION
**Pull request by teoli2003**
Most were red links to a non-existent glossary entry that never existed.

The wikipedia page is of good quality, let's link to it instead.

### CODE REVIEW

File: files/en-us/web/api/htmlanchorelement/hash/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The
 string containing a `'#'` followed by the fragment
 identifier of the URL.
 
-The fragment is not [percent-decoded](/en-US/docs/Glossary/percent-encoding). If the URL does not
+The fragment is not [percent-decoded](https://en.wikipedia.org/wiki/URL_encoding). If the URL does not
```
Comment (hamishwillee) [inline files/en-us/web/api/htmlanchorelement/hash/index.md:15]: Agree with this link change. But should we change the link text to match? URL encoding makes more sense to me that "percent encoding". (and of course I mean "in all these cases")

File: files/en-us/web/javascript/reference/global_objects/escape/index.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ A new string in which certain characters have been escaped.
...
 
-> **Note:** This function was used mostly for URL encoding and is partly based on the escape format in {{rfc(1738)}}. The escape format is _not_ an [escape sequence](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#escape_sequences) in string literals. You can replace `%XX` with `\xXX` and `%uXXXX` with `\uXXXX` to get a string containing actual string-literal escape sequences.
+> **Note:** This function was used mostly for [URL encoding](https://en.wikipedia.org/wiki/URL_encoding) and is partly based on the escape format in {{rfc(3986)}}. The escape format is _not_ an [escape sequence](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#escape_sequences) in string literals. You can replace `%XX` with `\xXX` and `%uXXXX` with `\uXXXX` to get a string containing actual string-literal escape sequences.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/escape/index.md:40]: This is specifically mentioned in the spec: https://tc39.es/ecma262/#sec-escape-string

> This encoding does not reflect changes to RFC 1738 made by RFC 3986.

Comment (teoli2003) [reply files/en-us/web/javascript/reference/global_objects/escape/index.md:40]:
```suggestion
> **Note:** This function was used mostly for [URL encoding](https://en.wikipedia.org/wiki/URL_encoding) and is partly based on the escape format in {{rfc(1738)}}. The escape format is _not_ an [escape sequence](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#escape_sequences) in string literals. You can replace `%XX` with `\xXX` and `%uXXXX` with `\uXXXX` to get a string containing actual string-literal escape sequences.
```

File: files/en-us/web/javascript/reference/global_objects/unescape/index.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ A new string in which certain characters have been unescaped.
...
 
-> **Note:** This function was used mostly for URL encoding and is partly based on the escape format in {{rfc(1738)}}. The `unescape()` function does _not_ evaluate [escape sequences](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#escape_sequences) in string literals. You can replace `\xXX` with `%XX` and `\uXXXX` with `%uXXXX` to get a string that can be handled by `unescape()`.
+> **Note:** This function was used mostly for [URL encoding](https://en.wikipedia.org/wiki/URL_encoding) and is partly based on the escape format in {{rfc(3986)}}. The `unescape()` function does _not_ evaluate [escape sequences](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#escape_sequences) in string literals. You can replace `\xXX` with `%XX` and `\uXXXX` with `%uXXXX` to get a string that can be handled by `unescape()`.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/unescape/index.md:40]:
```suggestion
> **Note:** This function was used mostly for [URL encoding](https://en.wikipedia.org/wiki/URL_encoding) and is partly based on the escape format in {{rfc(1738)}}. The `unescape()` function does _not_ evaluate [escape sequences](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#escape_sequences) in string literals. You can replace `\xXX` with `%XX` and `\uXXXX` with `%uXXXX` to get a string that can be handled by `unescape()`.
```

File: files/en-us/web/api/htmlanchorelement/hash/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/htmlanchorelement/hash/index.md]: @hamishwillee You are right. I uniformized it.

**Review by hamishwillee — APPROVED** [review]
Thanks @teoli2003 !

**Pull request merged**